komodo/release_transpiler.py

```diff
@@ -204,20 +204,31 @@ def main():
     matrix_parser.add_argument(
         "--release-folder",
         required=True,
+        type=dir_path,
         help="Folder with existing release file (default: None)",
     )
     matrix_parser.add_argument(
         "--override-mapping",
         required=True,
+        type=check_if_valid_file,
```

Comment:
since this is a "type", should we rename from `check_if_valid_file` to simply `valid_file`?

Reply:
Good idea!